src/QirRuntime/lib/qdk/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+#===============================================================================
+# Produce the qdk dynamic library
+#
+add_library(qdk SHARED
+  qdk.cpp
+  $<TARGET_OBJECTS:qir-rt-support-obj>
+  $<TARGET_OBJECTS:qir-qis-support-obj>
+  $<TARGET_OBJECTS:simulators-obj>

```

Comment:
Just as a side note to validate my own understanding, at this point the `add_subdirectory(Simulators)` directive from earlier has run such that simulators have been built by this point in the make? d496b03a25428e91c10e14450c34208823bd3fc6 from the **experimental** open systems simulator / QIR integration branch depends on that, so just wanted to make sure I understood. Thanks for the help!

Reply:
Yes, I believe, you understanding is correct.